src/main/java/org/folio/circulation/resources/ScheduledNoticeProcessingResource.java

```diff
@@ -2,57 +2,59 @@
 
 import static org.folio.circulation.support.Result.failed;
 
+import java.util.Collection;
+import java.util.concurrent.CompletableFuture;
+
 import org.folio.circulation.domain.MultipleRecords;
-import org.folio.circulation.domain.notice.schedule.ScheduledDueDateNoticeHandler;
+import org.folio.circulation.domain.notice.schedule.ScheduledNotice;
 import org.folio.circulation.domain.notice.schedule.ScheduledNoticesRepository;
 import org.folio.circulation.support.Clients;
 import org.folio.circulation.support.NoContentResult;
 import org.folio.circulation.support.ResponseWritableResult;
 import org.folio.circulation.support.Result;
 import org.folio.circulation.support.RouteRegistration;
 import org.folio.circulation.support.http.server.WebContext;
-import org.joda.time.DateTime;
-import org.joda.time.DateTimeZone;
 
 import io.vertx.core.http.HttpClient;
 import io.vertx.ext.web.Router;
 import io.vertx.ext.web.RoutingContext;
 
-public class ScheduledNoticeProcessingResource extends Resource {
+public abstract class ScheduledNoticeProcessingResource extends Resource {
 
-  private static final int SCHEDULED_NOTICES_PROCESSING_LIMIT = 100;
```

Comment:
Has the processing limit moved or been removed?

Reply:
ScheduledNoticeProcessingResource became abstract and the constant was moved to DueDateScheduledNoticeProcessingResource.